org.adempiere.base/src/org/compiere/model/MCost.java

```diff
@@ -1428,6 +1494,36 @@ public static MCost get (Properties ctx, int AD_Client_ID, int AD_Org_ID, int M_
 		int M_AttributeSetInstance_ID,
 		String trxName)
 	{
+		return get (ctx, AD_Client_ID, AD_Org_ID, M_Product_ID, M_CostType_ID, C_AcctSchema_ID, M_CostElement_ID, M_AttributeSetInstance_ID, null, null, trxName);
+	}
+	
+	/**
+	 * Get Cost Record
+	 * @param ctx context
+	 * @param AD_Client_ID client
+	 * @param AD_Org_ID org
+	 * @param M_Product_ID product
+	 * @param M_CostType_ID cost type
+	 * @param C_AcctSchema_ID as
+	 * @param M_CostElement_ID cost element
+	 * @param M_AttributeSetInstance_ID asi
+	 * @param dateAcct account date

```

Comment:
should clarify this is optional and is use to retrieve cost history record

Reply:
Done